action.yml

```diff
@@ -27,6 +27,9 @@ inputs:
   fetch-depth:
     description: 'Number of commits to fetch. 0 indicates all history for all branches and tags.'
     default: 1
+  lfs:
+    description: 'Whether to download Git-LFS files'
+    default: ''
```

Comment:
falsy string instead of `false` because according to the json scheme these default values should all be strings (there are warning for `fetch-depth` and the above values)